static/images/svg-icons/rum.svg

```diff
@@ -1,20 +1,15 @@
-<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 42 42">
-  <defs>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512">
+    <path d="M556.785 79.19v354.149a44.4 44.4 0 0 1-13.978 33.028 46.638 46.638 0 0 1-33.867 13.624H67.064q-19.353 0-33.868-13.624-14.49-13.659-13.96-33.028V79.19A44.446 44.446 0 0 1 33.2 46.162 48.687 48.687 0 0 1 67.064 32H508.94a45.786 45.786 0 0 1 33.867 14.162c9.307 9.408 13.978 20.445 13.978 33.028zm-388.924 8a16.632 16.632 0 0 0 16.764 16.5h.035a16.631 16.631 0 0 0 16.8-16.462v-.035a16.8 16.8 0 0 0-33.6-.035zm-50.4 0a16.633 16.633 0 0 0 16.765 16.5h.035a16.632 16.632 0 0 0 16.8-16.462v-.035a16.631 16.631 0 0 0-16.764-16.5h-.035a16.633 16.633 0 0 0-16.8 16.462zm-50.4 0a16.632 16.632 0 0 0 16.764 16.5h.035a16.631 16.631 0 0 0 16.8-16.462v-.035A16.632 16.632 0 0 0 83.9 70.689h-.035a16.631 16.631 0 0 0-16.8 16.462zm451.569 66.963a8.368 8.368 0 0 0-3.225-6.821 10.245 10.245 0 0 0-6.468-2.62H67.064a8.9 8.9 0 0 0-6.468 2.62 11.355 11.355 0 0 0-3.225 6.821v279.135a7.924 7.924 0 0 0 3.225 6.3 13.841 13.841 0 0 0 6.468 3.142H508.94a8.251 8.251 0 0 0 6.468-3.142 13.4 13.4 0 0 0 3.225-6.283zM370.143 310.115L236.1 175.8v183.672l41.646-40.2 38.907 92.528 30.273-12.7-37.426-88.935h60.629v-.05z"/>
+<defs>
     <style>
       .cls-1 {
         fill-rule: evenodd;
         fill: url(#linear-gradient);
       }
     </style>
-    <linearGradient id="linear-gradient" x1="1.11772" y1="20.68508" x2="40.61532" y2="20.68508" gradientUnits="userSpaceOnUse">
-      <stop offset="0" stop-color="#d671d0"/>
-      <stop offset="1" stop-color="#7a71cb"/>
+    <linearGradient id="paint0_linear5" x1="0.855469" y1="15.8637" x2="14.0857" y2="-1.67875" gradientUnits="userSpaceOnUse">
```

Comment:
The current stops produce a hard break between the colors:

<img width="64" alt="Screen Shot 2019-12-04 at 9 45 19 PM" src="https://user-images.githubusercontent.com/1048442/70200187-a468b680-16e1-11ea-8aaf-a7f92db4ceff.png">

For a smooth gradient instead, I'd recommend this:

```suggestion
    <linearGradient id="paint0_linear5" x1="20%" y1="100%" x2="100%" y2="0%" gradientUnits="userSpaceOnUse">
```

which gives you this:

<img width="64" alt="Screen Shot 2019-12-04 at 10 37 17 PM" src="https://user-images.githubusercontent.com/1048442/70202121-ae40e880-16e6-11ea-85ce-42b65ba6dec3.png">


The gradient `x` and `y` values for all the other icons are defined in pixels, but no idea how/where they were generated. Using these percentage values here produces essentially the same visual effect as the others. (It's also way more future-proof, as it's resilient to any change in icon sizes.)